docs/tutorial/part-eight/index.md

```diff
@@ -331,44 +330,45 @@ export const query = graphql`
 
-## Keep making it better
+## इसको बेहतर बनाते जाइये
 
-In this section, we've shown you a few Gatsby-specific tools to improve your site's performance and prepare to go live.
+इस भाग मे, हमने आपको कुछ Gatsby-स्पेसिफिक टूल्स दिखाए जो आपके साइट की प्रदर्शन को बढ़ाता है और आपके साइट को लाइव लेजाने मे आपकी मदद करता है। 
 
-Lighthouse is a great tool for site improvements and learning -- Continue looking through the detailed feedback it provides and keep making your site better!
+साइट मे सुधार लाने के लिए और सीखने के लिए Lighthouse एक बहुत उपयोगी साधन है -- इसमें बताये गए विस्तृत प्रतिक्रिया को देखना जारी रखिये और अपने साइट को बेहतर बनाते जाइये!
 
-## Next Steps
+## अगले कदम
 
-### Official Documentation
+### आधिकारिक डॉक्यूमेंटेशन
 
-- [Official Documentation](https://www.gatsbyjs.org/docs/): View our Official Documentation for _[Quick Start](https://www.gatsbyjs.org/docs/quick-start/)_, _[Detailed Guides](https://www.gatsbyjs.org/docs/preparing-your-environment/)_, _[API References](https://www.gatsbyjs.org/docs/gatsby-link/)_, and much more.
+- [आधिकारिक डॉक्यूमेंटेशन](https://www.gatsbyjs.org/docs/): हमारे आधिकारिक डॉक्यूमेंटेशन को देखिये _[जल्द शुरुआत](https://www.gatsbyjs.org/docs/quick-start/)_, _[विस्तृत गाइड्स](https://www.gatsbyjs.org/docs/preparing-your-environment/)_, _[API संधर्बे](https://www.gatsbyjs.org/docs/gatsby-link/)_, और बहुत कुछ चीज़ो के लिए।
 
-### Official Plugins
+### आधिकारिक प्लगिन्स
 
-- [Official Plugins](https://github.com/gatsbyjs/gatsby/tree/master/packages): The complete list of all the Official Plugins maintained by Gatsby.
+- [आधिकारिक प्लगिन्स](https://github.com/gatsbyjs/gatsby/tree/master/packages): सारे आधिकारिक प्लगिन्स की पूरी सूची जो की Gatsby द्वारा बनाये रखे गए है।
 
-### Official Starters
+### आधिकारिक स्टार्टर्स
 
-1.  [Gatsby's Default Starter](https://github.com/gatsbyjs/gatsby-starter-default): Kick off your project with this default boilerplate. This barebones starter ships with the main Gatsby configuration files you might need. _[working example](http://gatsbyjs.github.io/gatsby-starter-default/)_
-2.  [Gatsby's Blog Starter](https://github.com/gatsbyjs/gatsby-starter-blog): Gatsby starter for creating an awesome and blazing-fast blog. _[working example](http://gatsbyjs.github.io/gatsby-starter-blog/)_
-3.  [Gatsby's Hello-World Starter](https://github.com/gatsbyjs/gatsby-starter-hello-world): Gatsby Starter with the bare essentials needed for a Gatsby site. _[working example](https://gatsby-starter-hello-world-demo.netlify.com/)_
+1.  [Gatsby की ब्लॉग स्टार्टर](https://github.com/gatsbyjs/gatsby-starter-default): इस बॉयलरप्लेट के साथ अपने प्रोजेक्ट की शुरुआत कीजिये।
+यह बारबोनेस स्टार्टर, Gatsby के मुख्य कॉन्फ़िगरेशन फाइल्स के साथ शिप किया जाता है जिसकी आपको ज़रूरत पड़ सकती है। _[काम करता हुआ उदाहरण](http://gatsbyjs.github.io/gatsby-starter-default/)_
```

Comment:
remove line break here at line 358

Reply:
Hey, that's is not a line break, it is just too long to fit into the same line.

Reply:
Yes. right. 
Sorry, my bad.